docs/hsi-tests.d/org.fwupd.hsi.Amd.PlatformRollbackProtection.json

```diff
@@ -3,7 +3,10 @@
 	"name": "AMD Secure Processor Rollback protection",
 	"description": [
 		"AMD SOCs include the ability to prevent a rollback attack by a rollback protection feature on the secure processor.",
-		"This feature prevents an attacker from loading an older firmware onto the part after a security vulnerability has been fixed."
+		"This feature prevents an attacker from loading an older firmware onto the part after a security vulnerability has been fixed.",
+		"",
```

Comment:
not the empty string -- maybe this should live in `more-information` instead?

Reply:
Good idea; moved.